Dockerfile

```diff
@@ -138,13 +151,15 @@ ENV HZ_CLIENT_MODE=false \
 RUN  mkdir -p /opt/pega/kafkadata && \
      chgrp -R 0 /opt/pega/kafkadata && \
      chmod -R g+rw /opt/pega/kafkadata && \
+     chmod -R o+rwx /opt/pega/kafkadata && \
```

Comment:
Why are we granting other execute on kafkadata?  This doesn't seem like a good idea.

Reply:
kafkadata has subdirs, tomcat user will need to create them